code/modules/reagents/reagent_containers/food/drinks.dm

```diff
@@ -264,9 +267,12 @@
 
 /obj/item/weapon/reagent_containers/food/drinks/proc/imbibe(mob/user) //Drink the liquid within
 
-	user << "<span  class='notice'>You swallow a gulp of \the [src].</span>"
+	user << "<span  class='notice'>You swallow a gulp of \the [src].[lit ? " It's hot!" : ""]</span>"
 	playsound(user.loc,'sound/items/drink.ogg', rand(10,50), 1)
 
+	if(lit)

```

Comment:
There's been issues recently with silicons taking damage from manual temperature increases.
I think you should do
`if(lit && !issilicon(user))`
And we'll replace it with a mob proc at a later time.